packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -31,9 +31,22 @@ import 'theme.dart';
 ///
 /// {@tool dartpad}
 ///
-///
 /// ** See code in examples/api/lib/material/reorderable_list/reorderable_list_view.0.dart **
-///{@end-tool}
+/// {@end-tool}
+///
+/// Default [proxyDecorator] can overriden to customize an item when it is being dragged.
```

Comment:
This example demonstrates using the [proxyDecorator] callback to customize the appearance of a list item while it's being dragged.

Reply:
Thanks